modules/ROOT/pages/api-sync.adoc

```diff
@@ -1,71 +1,50 @@
-= API Sync
+= Sync Your API Projects with API Sync
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: api, anypoint platform, sync, api sync
+:keywords: api, anypoint platform, sync, api sync, MuleSoft VCS
```

Comment:
Is API Sync the new name for MuleSoft VCS?

Reply:
Oh, nm. I guess not.

Reply:
No. API Sync is the features Studio has to connect with MuleSoft VCS or other repos.